oelgoubi/elhura.store
# Feature/handle signup requests

**Functionalities or bugs :** 
  - Redesign of the signup form
  - Console.log of user informations after signup

**Tests case :** 
  - Go to Menu > Create an account (Créer un compte).
  - Start signing up : 
    - Make sure you are not able to click on the "Join" button if any of field is empty.
    - Make sure errors are showed if an invalid email format or passwords unmatch occur.
    - In a real case scenario, clicking the "Join" button should start registering email and password in the database (Not implemented yet) and sending confirmation email to the user email (not implemented yet).
    - A new form is showed to make choose between signing up as Customer, as Seller Company or as Admin.
    - Then a new form is showed to complete detailed informations.
    - Clicking the "Join" button should show logs in the browser console : 
      - Here again, we should register these detailed informations in the database in a real case scenario.
